Documentation/GettingStarted.md

```diff
@@ -54,9 +57,112 @@ Add a user to the libary. The user can be of any class, as long the calls implem
         NSLog(@"Failed to create Account: %@", error);
     }
 ```
+### 4. Inbound calls
 
-#### 4. Set callback for incoming calls
+#### 1. AppDelegate - instantiate CallKitProviderDelegate to a private property 
```

Comment:
You are right but it is not correct for documentation. I wanted to say that private property it better solution for us, not for people.